client/gutenberg/extensions/tiled-gallery/editor.js

```diff
@@ -9,7 +9,6 @@ import { createBlock, registerBlockType } from '@wordpress/blocks';
 /**
  * Internal dependencies
  */
-import './style.scss';
```

Comment:
We don't need to import this here at all because we already import it in `view.js` and it'll thus be loaded in the editor, too.